packages/app/obojobo-repository/shared/components/repository-nav.jsx

```diff
@@ -49,7 +77,12 @@ const RepositoryNav = props => {
 				{props.userId !== 0 ? (
 					<div className="repository--nav--current-user">
 						<button onClick={onToggleMenu}>
-							<div className="repository--nav--current-user--name">{props.displayName}</div>
+							<div onClick={onNotifications} className="repository--nav--current-user--name">
+								{props.displayName}
+
+								{notificationsExist && <div className="notification-indicator"></div>}
+							</div>
+
```

Comment:
I see what you're going for, but I think rather than make the entire username clickable all the time it would be better if the notification indicator was more prevalent, and the notification indicator was listening for clicks rather than the entire username/avatar area. You'll notice how the addition of this `onClick` has altered numerous unrelated snapshot files.

It may be better after all to go with the second option, where notifications are indicated with a clickable '# New Messages' element that appears under the username instead of inline.